include/st_pipeline_api.h

```diff
@@ -412,6 +414,12 @@ enum st22_quality_mode {
  * st20_rx_get_ext_frame.
  */
 #define ST20P_RX_FLAG_EXT_FRAME (MTL_BIT32(2))
+/**
+ * Flag bit in flags of struct st20p_rx_ops.
+ * Only used for internal convert mode.
+ * Perform the color format conversion on each packet.
```

Comment:
nit: add note it only support limited formats. Now only ST_FRAME_FMT_YUV422PLANAR10LE,  ST_FRAME_FMT_*** and others.